### DISCUSSION TITLE
Parameter `max_concurrent_reader_users`

### DISCUSSION THREAD

**Discussion by ktnr** [valhalla/valhalla#4681] [General]
What does the parameter `max_concurrent_reader_users` do? Could not find anything the docs. Could it be related to https://github.com/gis-ops/docker-valhalla/issues/81#issuecomment-2046129267? Does it only have an effect on tile generation, since it is part of Mjolnir, and has no effect on routing?

**Comment by nilsnolde** [comment]
if you follow that parameter around, it's only being used for the case when you want to load graph tiles from a remote file system via curl. it's just determining the size of the connection pool it seems.
